### ISSUE TITLE
Dark Theme mismatch

### ISSUE THREAD

**Issue by Mohitmadhav** [Project-Easter/Flutter-UI#80]
## Description
- The Dark theme color schemes are not matching correctly.
- Design: https://www.figma.com/file/uhWQuE4EKDHR8AqH4p3eFK/Books-App?node-id=0%3A1

## Screenshots
![image](https://user-images.githubusercontent.com/59333817/135658187-57650735-f33d-4cc0-ac6e-d03c24ccbb5f.png)
![image](https://user-images.githubusercontent.com/59333817/135658243-e1703308-cb0d-437a-9c1b-3095008584bc.png)
![image](https://user-images.githubusercontent.com/59333817/135658257-cb2eb90f-5e2e-488c-ac38-b4e474846316.png)

## Steps to Reproduce
You can change the color schemes according to Figma design

1. You can add a few colors by yourself if something is in your mind.
2.  Path: lib\common\themes.dart
3. Check for more such mismatches in other files.

## Additional data
Feel free to discuss your design ideas here.

**Comment by phan-dev** [comment]
@Mohitmadhav 
Can you assign this issues for me?

**Comment by VidhiBhatt01** [comment]
I can fix this, can you assign it to me?

**Comment by Mohitmadhav** [comment]
@VidhiBhatt01, I have an issue based on the light theme, you can work on that if you want. Also, more issues beginner-friendly issues will be created.

Till then @phan-dev, I'm assigning this issue to you.

**Comment by VidhiBhatt01** [comment]
Yeah I can work on that as well! No worries.. Also I will keep on checking more issues you create

**Comment by Mohitmadhav** [comment]
@phan-dev is it resolved?

**Comment by Mohitmadhav** [comment]
@phan-dev since you are not following up, I'll assign it to someone else.

**Comment by Mohitmadhav** [comment]
Cool @AryanKhubchandani. You can delete your previous branch and make a new one containing the updated changes.
Then maybe the personalize button may work for you.
